README.md

```diff
@@ -115,3 +115,34 @@ The base technologies are JavaScript, HTML and CSS. The frontend leverages [Reac
 | POST   | `/meetings`                     | `meeting_topic`,`meeting_date`, `meeting_time`,`host_id`,`attendee_id`,           | `create a new meeting`                            |
 | PUT    | `/meetings/:meeting_id`         | `meeting_topic` or `meeting_date` or `meeting_time` or `host_id` or `attendee_id` | `update a meeting by meeting_id,`                 |
 | DELETE | `/meetings/:meeting_id`         | -                                                                                 | `delete meeting by meeting_id`                    |
+
+## Intakes
+
+###### Mentor intake schema:
+
+  {
+    profile_id: '00u13oned0U8XP8Mb4x7',
+    email: 'fakeemail2@gmail.com',
+    location: 'Bumville, USA',
+    name: 'Hotdog Jeopardy',
```

Comment:
Just wanted to drop a note here to say that these example names are something else! 😂